### ISSUE TITLE
fsls produces a flat output instead of a tree structure in Windows

### ISSUE THREAD

**Issue by hyperlink** [cpsubrian/node-fsls#2]
Test output on Windows 7

```

  tree
    1) generates the nodes tree
    2) respects depth option
    3) respects structure option


  0 passing (27ms)
  3 failing

  1) tree generates the nodes tree:
     Uncaught AssertionError: {"nodes":["more/","more/bar.txt","more/deeper/","more/deeper/baz.js","more/foo.js","test.js"]} deepEqual {"nodes":[{"label":"more/","nodes":
["bar.txt","foo.js",{"label":"deeper/","nodes":["baz.js"]}]},"test.js"]}
      at C:\Users\Xiaoxin\Documents\GitHub\node-fsls\test\basic.js:7:14
      at C:\Users\Xiaoxin\Documents\GitHub\node-fsls\fsls.js:22:5
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:90:7)
      at Glob.EventEmitter.emit (events.js:95:17)
      at Glob._processEmitQueue (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:294:10)
      at Glob.emitMatch (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:271:10)
      at Glob._finish (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:232:8)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:173:27)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:430:32)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:430:32)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:430:32)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:397:14)
      at Glob._afterReaddir (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:642:17)
      at ReaddirReq.Req.done (C:\Users\Xiaoxin\AppData\Roaming\npm\node_modules\mocha\node_modules\glob\node_modules\graceful-fs\graceful-fs.js:142:5)
      at ReaddirReq.done (C:\Users\Xiaoxin\AppData\Roaming\npm\node_modules\mocha\node_modules\glob\node_modules\graceful-fs\graceful-fs.js:89:22)
      at Object.oncomplete (fs.js:107:15)

  2) tree respects depth option:
     Uncaught AssertionError: {"nodes":["more/","more/bar.txt","more/deeper/","more/deeper/baz.js","more/foo.js","test.js"]} deepEqual {"nodes":[{"label":"more/"},"test.j
s"]}
      at C:\Users\Xiaoxin\Documents\GitHub\node-fsls\test\basic.js:32:14
      at C:\Users\Xiaoxin\Documents\GitHub\node-fsls\fsls.js:22:5
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:90:7)
      at Glob.EventEmitter.emit (events.js:95:17)
      at Glob._processEmitQueue (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:294:10)
      at Glob.emitMatch (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:271:10)
      at Glob._finish (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:232:8)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:173:27)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:430:32)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:430:32)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:430:32)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:397:14)
      at Glob._afterReaddir (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:642:17)
      at ReaddirReq.Req.done (C:\Users\Xiaoxin\AppData\Roaming\npm\node_modules\mocha\node_modules\glob\node_modules\graceful-fs\graceful-fs.js:142:5)
      at ReaddirReq.done (C:\Users\Xiaoxin\AppData\Roaming\npm\node_modules\mocha\node_modules\glob\node_modules\graceful-fs\graceful-fs.js:89:22)
      at Object.oncomplete (fs.js:107:15)

  3) tree respects structure option:
     Uncaught AssertionError: {"nodes":[]} deepEqual {"nodes":[{"label":"more/","nodes":[{"label":"deeper/"}]}]}
      at C:\Users\Xiaoxin\Documents\GitHub\node-fsls\test\basic.js:47:14
      at C:\Users\Xiaoxin\Documents\GitHub\node-fsls\fsls.js:22:5
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:90:7)
      at Glob.EventEmitter.emit (events.js:95:17)
      at Glob._processEmitQueue (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:294:10)
      at Glob.emitMatch (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:271:10)
      at Glob._finish (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:232:8)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:173:27)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:430:32)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:430:32)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:485:34)
      at Glob.cb (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:315:11)
      at Glob.<anonymous> (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:355:16)
      at Glob._afterStat (C:\Users\Xiaoxin\Documents\GitHub\node-fsls\node_modules\glob\glob.js:561:6)
      at Object.oncomplete (fs.js:107:15)
```

**Comment by cpsubrian** [comment]
Thanks for the report. I'll take a look at the PR as soon as I have a minute.

**Comment by hyperlink** [comment]
Thanks for the quick response Brian. Just a tiny change. Glob takes only a forward slash even in Windows.

**Comment by cpsubrian** [comment]
Fixed with merge of #3
